docs/windows/setup.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Debugging on Windows"
+description: "This article explains how to configure Windows 10 and Windows 11 for .NET MAUI app deployment and debugging."
+ms.date: 02/28/2022
+---
+
+# Deploy and debug your .NET MAUI app on Windows
+
+You can use your local Windows development computer to deploy and debug a .NET MAUI app. This article describes how to configure Windows to debug a .NET MAUI app.
+
+[!INCLUDE [docs under construction](~/includes/preview-note.md)]
+
+## Configure Windows
+
+You must enable Developer Mode in Windows. Both Windows 10 and Windows 11 are supported.
+
+:::image type="content" source="media/setup/developer-mode-win11.png" alt-text="Developer mode settings in Windows 11 for MAUI .NET Windows app.":::
+
+### Windows 11
+
+Developer Mode is enabled in **Settings** app, under **Privacy & security** > **For developers**. To enable Developer Mode in Windows 11:
+
+01. Open the Start Menu.
+01. Type **Developer settings** in the search box and select it.
+01. Turn on **Developer Mode**.
+01. If you receive a warning message about Developer Mode, read it, and select **Yes** if you understand the warning.
+
+### Windows 10
+
+Developer Mode is enabled in **Settings** app, under **Update & Security** > **For developers**. To enable Developer Mode in Windows 10:
+
+01. Open the Start Menu.
+01. Search for **Developer settings**, select it.
+01. Turn on **Developer Mode**.
+01. If you receive a warning message about Developer Mode, read it, and select **Yes** if you understand the warning.
+
+## Target Windows
+
+In Visual Studio, set the **Debug Target** to **Framework (...)** > **net6.0-windows10.0.19041**:
```

Comment:
Maybe add a Todo because this build version is expected to change by GA.

Reply:
It's really dumb to have the version in there ugh. I'm adding some info about it and simplifying the item text. The screenshot provides the full experience and hopefully the combination of the text + screenshot will prevent people from being confused.